Cartfile.resolved

```diff
@@ -1,4 +1,3 @@
-github "daltoniam/Starscream" "3.0.4"
-github "daltoniam/common-crypto-spm" "1.1.0"
-github "daltoniam/zlib-spm" "1.1.0"
-github "watson-developer-cloud/swift-sdk" "v0.21.0"
+github "daltoniam/Starscream" "3.0.5"
+github "watson-developer-cloud/restkit" "2.0.0"
```

Comment:
You can remove Starscream and restkit, since the "swift-sdk" will download those dependencies with the correct versions.